cppcache/src/DistributedSystemImpl.cpp

```diff
@@ -143,11 +147,24 @@ void DistributedSystemImpl::unregisterCliCallback(int appdomainId) {
   }
 }
 
-void DistributedSystemImpl::setThreadName(const std::string& threadName) {
+std::string DistributedSystemImpl::getThreadName(std::thread::id id) {
```

Comment:
Is there any particular reason why you would want to obtain the thread name from another thread which is not the caller thread? Because if there is no reason, I'd say it'd be best to get the thread ID of the current thread within the function, and that would make syntax a little bit simpler.

Reply:
Good catch! You're right. Don't need to pass in threadId.